synapse/lib/storm.py

```diff
@@ -202,6 +202,7 @@
         'view': {'type': 'string', 'pattern': s_config.re_iden},
         'user': {'type': 'string', 'pattern': s_config.re_iden},
         'iden': {'type': 'string', 'pattern': s_config.re_iden},
+        'enabled': {'type': 'boolean', 'default': True},
```

Comment:
This is helping for future-self when we have enabled/disabled cruds for individual dmons.